docker-compose/cassandra-3.11/start_cass_311.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
```

Comment:
Newbie question: does this refer to `docker-compose.yml` via naming convention (defaulting)? Or how is the dependency indicated?

Reply:
yes, that's correct. `docker-compose.yml` is the default name.